README.md

```diff
@@ -42,7 +41,7 @@ Also, see the [release notes](http://github.com/adobe/brackets/wiki/Release-Note
 for a list of new features and known issues in each build.
 
 In addition to the core features built into Brackets, there is a large and growing
-community of developers building extensions that add all sorts of useful functionality.
+the community of developers building extensions that add all sorts of useful functionality.
```

Comment:
Bad change. New text now reads: "... there is a large and growing _the_ community of developers..."